nodes/nomos-node-api/src/http/backend/axum.rs

```diff
@@ -256,3 +261,47 @@ where
         Err(e) => (StatusCode::INTERNAL_SERVER_ERROR, e.to_string()).into_response(),
     }
 }
+
+#[utoipa::path(
+    post,
+    path = "/mempool/add/tx",
+    responses(
+        (status = 200, description = "add transaction to the mempool"),
```

Comment:
We need to be consistent in descriptions. I have seen usually we capitalize them. Tiny details.